tensorboard/plugins/distribution/README.md

```diff
@@ -0,0 +1,46 @@
+# The Distribution Dashboard
+
+## Overview
+
+The distributions dashboard lets users
+
+*	Create line charts to display the change in the biases and weights of tensors over time.
+*	Visualize high level statistics, with each line on the graph representing a percentile in the distribution, and the closeness of those lines representing how tightly the variables are being packed into a normal distribution
+
+To use this dashboard, first collect data with summaries and tag it as a histogram, then lay out the UI.
+
+![The Distribution Dashboard](https://user-images.githubusercontent.com/46605868/78409608-168dd100-75d8-11ea-987e-122689d99252.png)
+
+## The problem it solves
+
+The distributions dashboard flattens the display in the histograms dashboard, allowing for easier identification of which percentile of the biases and weights require the most change. The dashboard also includes a transparent overlay of the previous layers results, making it easier to visualize just how much each layer improves results.
+
+## Collecting data
+In order to execute the distributions dashboard, users must first collect [data with summaries](https://www.tensorflow.org/api_docs/python/tf/summary). Both old and new style summaries can be accepted.
+
+Data must also be given the histogram tag in order to appear in the distributions dashboard. This tag may be included in the tensorboard callback command as shown here,
+
+* tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)
+
+As long as histogram_freq >= 1, the distribution/ histogram dashboards will automatically execute once tensorboard is called.
+
+## Example Code
+See the standard [tensorboard getting started code](https://www.tensorflow.org/tensorboard/get_started) for an example of collecting data with summaries and correctly setting the histogram variable
+
+## Example of the Distributions Dashboard UI-
+The code in the tensorboard getting started example produces this dashboard.
+
+![The Distributions Dashboard Example](https://user-images.githubusercontent.com/46605868/78409824-b0ee1480-75d8-11ea-86cf-ed3475cf32d9.png)
+
+Each drop down menu above represents a tensor, with each being named for what type of layer it is. The two above correspond to these lines in the example code:
```

Comment:
> Each drop down menu above

I can see why 'drop down menus' might apply, but I think the naming could be clearer.  Would it make sense to say "Each collapsible section above"?